src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -294,6 +314,23 @@ private static CommandLineProgram extractCommandLineProgram(final String[] args,
         return null;
     }
 
+    // Return a list of (usually Picard) classes which we want to mask from the list of runnable tools,
+    // usually because want to favor the like-named GATK tool. Any CommandLineProgram that shares a simple class
+    // name with a Picard tool must be on this list. Failure to mask it out will result in name collisions of
+    // simple class names in Main.
+    private static Set<String> getMaskedToolClasses() {
```

Comment:
Similarly to `getPackageList()` and `getClassList()`, this should be a protected non-static method, overridable but downstream projects, to provide a set of tools to remove from the command line. Downstream projects require more control on this.

On the other hand, I suggest to change the signature to return `List<Class<? extends CommandLineProgram>>`, but that should return a common interface for tools (both Picard and GATK).